GOVERNANCE.md

```diff
@@ -49,27 +52,30 @@ Collaborators may opt to elevate significant or controversial modifications to
 the TSC by assigning the `tsc-review` label to a pull request or issue. The
 TSC should serve as the final arbiter where required.
 
+See:
+
 * [Current list of Collaborators](./README.md#current-project-team-members)
 * [A guide for Collaborators](./COLLABORATOR_GUIDE.md)
 
 ### Collaborator Activities
 
 Typical activities of a Collaborator include:
 
-* helping users and novice contributors
-* contributing code and documentation changes that improve the project
-* reviewing and commenting on issues and pull requests
-* participation in working groups
-* merging pull requests
+* Helping users and novice contributors
+* Contributing code and documentation changes that improve the project
+* Reviewing and commenting on issues and pull requests
+* Participation in working groups
+* Merging pull requests
 
 The TSC periodically reviews the Collaborator list to identify inactive
 Collaborators. Past Collaborators are typically given _Emeritus_ status. Emeriti
 may request that the TSC restore them to active status.
 
 ## Technical Steering Committee
 
-The Technical Steering Committee (TSC) has final authority over this project
-including:
+A subgroup of the Collaborators form the Technical Steering Committee (TSC),
+who admits and oversees all top-level Projects in the Node.js Foundation,
```

Comment:
For clarity, maybe put a period instead of a comma after `(TSC)` on the line above this one? Then start this line with `The TSC admits and oversees all...`

There are currently no "top-level projects" other than core. (`express` and `libuv` never completed the process and therefore never graduated to top-level project status.) There is unlikely to ever be any other top-level projects. I think the terminology should be removed as it is confusing. Using it implies that that whole top-level project is a process that exists, a thing we do. I would just say:

> The TSC has final authority over this project, including:

Reply:
@Trott I copied the sentence from the TSC readme, we should also update that one.

Reply:
@joyeecheung Sounds like a good idea to me. https://github.com/nodejs/TSC/pull/477